server/src/immich/controllers/asset.controller.ts

```diff
@@ -145,4 +147,16 @@ export class AssetController {
   ): Promise<AssetResponseDto> {
     return this.service.update(authUser, id, dto);
   }
+
+  @Post('stack')
```

Comment:
These should go before the previous route, which is a wildcard `PUT asset/:id`.

Reply:
Handled.